src/main/frontend/components/query/result.cljs

```diff
@@ -0,0 +1,78 @@
+(ns frontend.components.query.result
```

Comment:
Had to extract this ns b/c node tests wouldn't let me depend on frontend.ui which has a require, react-tippy, which fails in node environments. I didn't modify anything in here except for introducing get-group-by-page and the bugs fixes for it